client/my-sites/plugins/access-control.js

```diff
@@ -48,4 +39,4 @@ const hasRestrictedAccess = ( site ) => {
 	}
 };
 
-export default { hasRestrictedAccess };
+export default { getHasRightsToManagePlutins, hasRestrictedAccess };
```

Comment:
Since you're being hazed to our way of working in Calypso, @jorgefilipecosta (:D), I would suggest changing this as Kerry pointed out and seeing how that affects consumers of the module.